package.json

```diff
@@ -18,6 +54,9 @@
     "url": "git+https://github.com/elpheria/rpc-websockets.git"
   },
   "dependencies": {
+    "@types/uuid": "^8.3.4",
+    "@types/ws": "^8.2.2",
+    "buffer": "^6.0.3",
```

Comment:
Downstream clients may actually need these dependencies if they're building for the browser, so I moved them out of `devDependencies`.